nixos/modules/services/databases/postgresql.nix

```diff
@@ -57,7 +57,10 @@ in
         type = types.path;
         example = "/var/lib/postgresql/11";
         description = ''
-          Data directory for PostgreSQL.
+          The data directory for PostgreSQL. If left as the default value
+          this directory will automatically be created before the PostgreSQL server starts, otherwise
+          the sysadmin is responsible for ensuring the directory exists with appropriate ownership
+          and permissions.
```

Comment:
Maybe explain what the default is (e.g. with `defaultText`)? The option doesn't have an option-default, so folks reading the manual will be instructed to use the default, but have to check the source to find out what it actually is.